stacks/core/README.md

```diff
@@ -50,9 +81,10 @@ This stack will be deployed with the basic services, which are related to the th
    docker stack deploy --compose-file connector.yml knot-cloud-core
    ```
 
-1. Enter connector directory and install its dependencies:
+1. Go to the connector directory, which is in the `<path>` directory, and install its dependencies:
 
    ```bash
+   cd ../knot-fog-connector
```

Comment:
Nit: I'd suggest following the pattern established on line 25:
```suggestion
   cd <path>/knot-fog-connector
```

Reply:
Done.